harness/determined/core/_checkpoint.py

```diff
@@ -414,7 +468,11 @@ def get_metadata(self, storage_id: str) -> Dict[str, Any]:
 
     @contextlib.contextmanager
     def store_path(
-        self, metadata: Optional[Dict[str, Any]] = None, *, shard: bool = False
+        self,
+        metadata: Optional[Dict[str, Any]] = None,
+        *,
+        shard: bool = False,
+        paths: Optional[storage.Paths] = None,
```

Comment:
I thought you said you did not modify store_path?

Reply:
and you are correct... that shouldn't be here. removing.